pywhy_graphs/algorithms/pag.py

```diff
@@ -908,3 +908,232 @@ def _check_ts_node(node):
         )
     if node[1] > 0:
         raise ValueError(f"All lag points should be 0, or less. You passed in {node}.")
+
+
+def orient_edges(graph: CPDAG) -> None:
```

Comment:
Probably don't just copy/paste the code directly. Prolly should be renamed, etc.

E.g. `orient_edges` is sort of a bad name, cuz it is in the `pag.py` file and it works on a CPDAG. Perhaps, you rename it to a private function called `_apply_meek_rules`

Reply:
Sure.